config/routes.rb

```diff
@@ -12,6 +12,12 @@
 
   get 'sentiment' => 'sentiment#check'
 
+  get 'favorite' => 'favorite#favorite'
+
+  post 'favorite/create/:company_id' => 'favorite#create_favorite', as: :create_favorite
```

Comment:
Ah okay. Sorry I misunderstood what was going on here, but after reading what you said and checking out the docs, I agree. Good code :)